.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +10,5 @@
 * [ ] Does your logo follow the [guidelines](https://github.com/NREL/mobility_landscape#logos)?
 * [ ] Does your project/product name match the text on the logo?
 * [ ] Have you ensured that all of your filter variables meet the standards of other entries?
+* [ ] Have you checked the pdf and png files to ensure that the landscape looks appropriate? To do so, go to Actions and then Artifacts, and download the png/pdf to see.
 * [ ] ~15 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?
```

Comment:
you need to remove this line. There is no CNCF-Bot and there is no staging server. Also, people should check the landscape and then add an "LGTM" to the bug like they would have done if there were a staging server.

We don't have a staging server.
Our staging server is the generated artifacts.